src/high_scores.rs

```diff
@@ -11,7 +12,7 @@ use std::io::Write;
 use std::time::Duration;
 
 // https://users.rust-lang.org/t/convert-box-dyn-error-to-box-dyn-error-send/48856/8
-type AnyErrorThreadSafe = Box<dyn std::error::Error + Send + Sync>;
+pub type AnyErrorThreadSafe = Box<dyn std::error::Error + Send + Sync>;
```

Comment:
Let's see if this is needed.

```suggestion
type AnyErrorThreadSafe = Box<dyn std::error::Error + Send + Sync>;
```